### PR TITLE
FEAT [base-structure]

### PR DESCRIPTION
**Pull request by SergeyZorych**
Inited base structure for project
Added methods for fetching categories

### CODE REVIEW

File: app/endpoints/v1/__init__.py [hunk 1]
```diff
@@ -0,0 +1,18 @@
+from fastapi import APIRouter
+
+router_v1 = APIRouter(
+    prefix="/multi/v1",
+    tags=["Routes v1"],
+    # dependencies=[Depends(auth_scheme)],
```
Comment (Andestrial123) [inline app/endpoints/v1/__init__.py:6]: Do u need this ?

File: config/config-local.yaml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+app:
+  debug: true
+
+logger:
+  level: DEBUG
+  base_name: multi-api
+
+db:
+  # docker
+  # host: db
```
Comment (Andestrial123) [inline config/config-local.yaml]: Do u need this commented code ? If not , remove commented code

File: app/endpoints/v1/__init__.py (hunk 1 above)
Comment (SergeyZorych) [reply app/endpoints/v1/__init__.py:6]: yes

Comment (SergeyZorych) [reply app/endpoints/v1/__init__.py:6]: for firebase auth in future

File: config/config-local.yaml (hunk 2 above)
Comment (SergeyZorych) [reply config/config-local.yaml]: yes, i need
because if you want to run it locally in docker you need to change host to db

Comment (Andestrial123) [reply config/config-local.yaml]: and u will always keep commented code for it ?

Comment (SergeyZorych) [reply config/config-local.yaml]: yes, until I add it into readme

Comment (Andestrial123) [reply config/config-local.yaml]: i think it's not good approach to keep commented code. U have git diff for it

**Review by Andestrial123 — APPROVED** [review]

**Pull request merged**